lib/swaps/Swaps.ts

```diff
@@ -369,7 +370,7 @@ class Swaps extends EventEmitter {
 
     const { makerCurrency, makerAmount, takerCurrency, takerAmount } = Swaps.calculateMakerTakerAmounts(quantity, price, isBuy, requestBody.pairId);
 
-    const swapClient = this.swapClients.get(takerCurrency);
+    const swapClient = this.swapClientManager.get(takerCurrency);
```

Comment:
Same here for try/catch if the swap client doesn't exist, replacing the if statement below.